lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -6,7 +6,7 @@ protocolParams:
   protocolVersion:
     minor: 0
     major: 0
-  decentralisationParam: 1
+  decentralisationParam: 1 # means 0% decentralised
```

Comment:
Weird that github shows this diff, whereas master `decentralisationParam` is actually `0.25` :thinking:

Reply:
Well github doesn't show merge conflicts. It can only show a diff to the merge base.